packages/nextjs/src/components/_common/productPanelExpandable/ProductPanelExpandable.module.scss

```diff
@@ -59,12 +58,6 @@
     :global(.navds-expansioncard__header-button) {
         align-self: center;
     }
-
-    @media #{common.$mq-screen-mobile} {
-        :global(.navds-expansioncard__header-button) {
-            transform: translateX(10px);
-        }
-    }
```

Comment:
Transform tuller det til med trikset vi bruker for å få større klikkflate, så jeg løste posisjonen til chevron på en annen måte.